app/assets/stylesheets/_orders.scss

```diff
@@ -73,3 +73,6 @@
     text-align:center;
   }
 }
+
+#order_filter_status {
```

Comment:
Empty rule<br>Avoid using id selectors<br>Selector `order_filter_status` should be written in lowercase with hyphens